File: aspnet-core/src/Zinlo.Application.Shared/Register User/Dto/PaymentDetails.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Zinlo.Register_User.Dto
{
   public class PaymentDetails
    {
        public string CardNumber { get; set; }
        public string CVVCode { get; set; }
        public DateTime ExpiryDate { get; set; }
        public string Email { get; set; }
        public int Commitment { get; set; }
    }
}

File: angular/src/account/Models/PaymentDetails.ts
export class PaymentDetails{
    cardNumber:string
    CVVCode:string
    expiryDate:string
    email:string
    commitment:string
    constructor()
    {
        this.cardNumber = '';
        this.CVVCode = "";
        this.expiryDate="";
        this.email="";
        this.commitment="";
    }
}